### DISCUSSION TITLE
Help - To Top button on EVERY page

### DISCUSSION THREAD

**Discussion by constantcopyandwork** [thuliteio/doks#752] [Q&A] [answered]
Hey, all, 

I've been using Hugo for only a little while now, but I really like it and this theme!

I do need help with getting a to top button to show up on every page.
I have all the codes and everything works fine on home page. I added it as a button and it needs to be in the body of the page. 

Any pointers on how to add it to ALL pages? Possibly as a partial? Adding it manually would be very time-consuming. 

Thanks!

**Comment by roheclassic** [comment, marked as answer]
Maybe you can achieve that with the following steps:

1. Create ```layouts/_default/``` directory.
2. Copy ```node_modules/@hyas/doks/layouts/_default/baseof.html``` to the above directory.
3. Place the button on ```baseof.html```.

For example:

```
    {{ block "sidebar-footer" . }}{{ end }}
    {{ partial "footer/footer.html" . }}
    {{ partial "footer/script-footer.html" . }}
    <button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>  <----*
  </body>
</html>
```

- I tried to implement it on my own site：
https://px.tarotcard.page/en/cards/major_arcana/1/


Hope this is useful for you.

**Reply by constantcopyandwork** [reply]
I was able to implement it by creating a new partial in `baseof.html`. Thanks for the suggestion!
